lib/sqlalchemy_ingres/base.py

```diff
@@ -802,6 +802,7 @@ def get_indexes(self, connection, table_name, schema=None, **kw):
             WHERE
                 i.index_name = c.index_name
             AND i.index_owner = c.index_owner
+            AND i.system_use = 'U'
```

Comment:
Code looks good, I think this should be conditional under user control.

Reply:
Thanks for the review and suggestion.

Added logic allowing inspection of system generated indexes by setting connection execution option `inspect_indexes="ALL"`. The default is to return only user-defined indexes.

Example program to check additional code: [inspectIndexes.py.txt](https://github.com/user-attachments/files/16317108/inspectIndexes.py.txt)

Output:

    C:\test\> python inspectIndexes.py MYURL1
    Inspect1: [{'name': 'address_idx', 'column_names': ['address_id'], 'unique': False}]
    Inspect2: [{'name': 'address_idx', 'column_names': ['address_id'], 'unique': False}, {'name': '$house_u000001eb00000000', 'column_names': ['house_id', 'address_id'], 'unique': True}]
    Inspect3: [{'name': 'address_idx', 'column_names': ['address_id'], 'unique': False}]